Formula/c/cpdf.rb

```diff
@@ -0,0 +1,43 @@
+class Cpdf < Formula
+  desc "PDF Command-line Tools"
+  homepage "https://github.com/johnwhitington/cpdf-source"
+  url "https://github.com/johnwhitington/cpdf-source/archive/refs/tags/v2.8.1.tar.gz"
+  sha256 "bdd7caf1e5e55e65e4ece96eeeb3e5894c195ca5a9a274ddc27ac50a321d5c75"
+  license "AGPL-3.0-only"
+
+  depends_on "ocaml-findlib" => :build
+  depends_on "camlpdf"
+  depends_on "ocaml"
+
+  def install
+    # For OCamlmakefile
+    ENV.deparallelize
+
+    # Work around for https://github.com/Homebrew/homebrew-test-bot/issues/805
+    if ENV["HOMEBREW_GITHUB_ACTIONS"] && !(Formula["ocaml-findlib"].etc/"findlib.conf").exist?
+      ENV["OCAMLFIND_CONF"] = Formula["ocaml-findlib"].opt_libexec/"findlib.conf"
+    end
+
+    ENV["OCAMLFIND_DESTDIR"] = lib/"ocaml"
+
+    (lib/"ocaml").mkpath
+    cp Formula["ocaml"].opt_lib/"ocaml/Makefile.config", lib/"ocaml"
+
+    # install in #{lib}/ocaml not #{HOMEBREW_PREFIX}/lib/ocaml
+    inreplace lib/"ocaml/Makefile.config" do |s|
+      s.change_make_var! "prefix", prefix
+    end
+
+    system "make"
+    (lib/"ocaml/stublibs").mkpath # `make install` assumes this directory exists
+    system "make", "install", "STDLIBDIR=#{lib}/ocaml"
+
+    rm lib/"ocaml/Makefile.config" # avoid conflict with ocaml
+
+    bin.install "cpdf"
+  end
+
+  test do
+    assert_match "cpdf AGPL Version 2.8.1 (28th March 2025)", shell_output(bin/"cpdf")
```

Comment:
```suggestion
    assert_match version.to_s, shell_output(bin/"cpdf")
```
You should also come up with a better test, see https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula
> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Reply:
Thanks. In the general case, I can see your point. For example, if an executable needs resource files and they have not been installed.

Cpdf, however, is an entirely self-contained executable. If it is capable of printing its version number, it works. If it isn't, it doesn't.